.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "4.0"
+      }
+    }]
+  ],
+  "plugins": [
+    "add-module-exports",
```

Comment:
Yup. It was for our `index.js` file. I've changed this file to avoid using this plugin. 👍